### ISSUE TITLE
If Lantern is unable to proxy traffic for a site, it should be added to the detour whitelist so that we don't proxy it

### ISSUE THREAD

**Issue by fffw** [getlantern/lantern#2915]
We see error `Bad status code on CONNECT response: 403 on pass 0...continuing`.

It's the same error as https://github.com/getlantern/lantern/issues/2395. As we already removed the country list, the cause might not be the same.

> ERROR balancer: balancer.go:99 Unable to dial tcp://mynamedomain.koko:443: Bad status code on CONNECT response: 403 on pass 0...continuing

A quick scan give me an impression that all those domains are not resolvable.

**Comment by myleshorton** [comment]
Yup they look like almost all at least very weird domains. Some are direct IP addresses, others look like they're likely DNS entries for localhost or something else odd like that. Not sure how to solve the latter.

**Comment by oxtoacart** [comment]
It seems that if Lantern encounters something that it definitely can't proxy, we should add it to the detour "whitelist".  I'm renaming the ticket to reflect that.
